src/pluto/pluto.py

```diff
@@ -189,27 +189,27 @@ def create_mismatch(df_mismatch, v1, v2, v3, condo, mapped):
 
             groups = [
                 {
-                    "title": "Mismatch graph -- finance",
+                    "title": "Mismatch graph -- finance fields",
```

Comment:
Can these titles be larger? I changed the `update_layout` call in in the for loop to 
```python
                fig.update_layout(
                    title={"text": group["title"], "font": dict(size=24)},
                    template="plotly_white",
                )
```
and thought it made the app easier to navigate

Reply:
Alternatively if you don't feel like iterating on this we could skip it, it's a minor aesthetic thing and it seems like we have more interesting work to in the second week of this sprint

Reply:
Agreed that we should do that in the future (although I'm honestly tempted by having no title, and using the subheader tag instead) but think its better left for a different issue. Happy to write it up - increased the size on these for now

Reply:
to your point though, i added a global color scheme constant as part of #113, could be same implementation